components/invitation_modal/invitation_modal_members_step.jsx

```diff
@@ -150,6 +151,7 @@ export default class InvitationModalMembersStep extends React.Component {
                             type='text'
                             readOnly={true}
                             value={inviteUrl}
+                            aria-label={localizeMessage('invitation_modal.members.share_link.input', 'team invite link')}
```

Comment:
Hey @jespino, I've seen both `localizeMessage` and `formatMessage` used in situations like this. Do you know why one would be used over another and/or which one is (more?)correct to use? I seem to recall a conversation about this but don't remember the outcome. Thx.

Reply:
The `localizeMessage` comes from our utils, and `formatMessage` from the `intl` context, the second is the most correct approach, so i going to change it.